pkg/controller/sparkapplication/controller_test.go

```diff
@@ -333,7 +333,7 @@ func TestShouldRetry(t *testing.T) {
 	}
 
 	restartPolicyAlways := v1beta1.RestartPolicy{
-		Type:                             v1beta1.Always,
+		Type: v1beta1.Always,
```

Comment:
The formatting seems off. Can you run `go fmt ./...`?

Reply:
done